docs/framework/whats-new/obsolete-members.md

```diff
@@ -318,7 +318,7 @@ The tables in this article list the type members that are obsolete in the
-|<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetDragDropCompletedEffects%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Consider using                                          <xref:System.Activities.Presentation.DragDropHelper.SetDragDropMovedViewElements%28System.Windows.DragEventArgs%2CSystem.Collections.Generic.IEnumerable%7BSystem.Activities.Presentation.WorkflowViewElement%7D%29> instead.|  
+|<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetDragDropCompletedEffects%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Consider using <!--zz <xref:System.Activities.Presentation.DragDropHelper.GetDragDropMovedViewElements%28System.Windows.DragEventArgs%2CSystem.Collections.Generic.IEnumerable%7BSystem.Activities.Presentation.WorkflowViewElement%7D%29> -->                                         `System.Activities.Presentation.DragDropHelper.GetDragDropMovedViewElements(System.Windows.DragEventArgs,System.Collections.Generic.IEnumerable{System.Activities.Presentation.WorkflowViewElement})` instead.|  
 |<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetDraggedModelItem%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Use                                          <xref:System.Activities.Presentation.DragDropHelper.GetDraggedModelItems%2A> instead.|  
 |<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetDroppedObject%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Use                                          <xref:System.Activities.Presentation.DragDropHelper.GetDroppedObjects%2A> instead.|  
 |<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.SetDragDropCompletedEffects%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Consider using                                          <!--zz <xref:System.Activities.Presentation.DragDropHelper.SetDragDropMovedViewElements%2A> --> `xref:System.Activities.Presentation.DragDropHelper.SetDragDropMovedViewElements` instead.|  
```

Comment:
The obsolete text should red, "This method does not support dragging multiple items."